### ISSUE TITLE
[REQUEST] Autocorrect misspelled words

### ISSUE THREAD

**Issue by ascworl** [florisboard/florisboard#1283]
Greetings
When typing misspelled words, it should auto correct to the proper spelling after pressing Space.
Other keyboards such as Openboard and Swiftkey have this feature. Although, Openboard spell check doesn't work in certain apps such as Simplenote. The suggestions are blank.

Is this currently possible? Thank you

**Comment by pslyman** [comment]
Agreed. This tends to disrupt the flow of typing as I have to select the correct word suggestion for each word instead of typing as I normally would.

**Comment by vulpes2** [comment]
This would be a very useful feature to have, it's something that I really miss from AnySoftKeyoard and Gboard. I think this should be enabled by default and the user may disable it if they don't like it.

**Comment by Henry-Hiles** [comment]
This is absolutely needed.

**Comment by adarsh1001** [comment]
A must have feature the lack of which is the only reason I keep switching back to OpenBoard even though it doesn't have glide gestures. Hope this gets implemented soon.

**Comment by FlorentLM** [comment]
Indeed, that makes the Florisboard unusable for me, and I have to use Gboard

**Comment by hk8607** [comment]
Oh lord yes.. if auto-correct is added I'd be 100% dedicated to it..

**Comment by K4LCIFER** [comment]
I would also like autocorrect. I cant really use the keyboard without it.

**Comment by bertin0** [comment]
It's on the roadmap and currently in development.

**Comment by Gitonthegrid** [comment]
totally agree. It is one of the best keyboards I have used, but I cant live without autocorrect (anymore). 
But I have to say that they keyboard is so good, that I have a pretty accurate typing. I wait till the next update where the autocorrect will work. 

And the dictionary also don`t have any function right now because the missing autocorrect feature right?

**Comment by heschy2** [comment]
> It's on the roadmap and currently in development.

Has there been any progress?  Anything in this issue is 1-3 Years old.  Is it still planned and just low-priority or has it been discarded?  What is the current status of this?

**Comment by Henry-Hiles** [comment]
I now use heliboard instead, it has all these nice features, plus a better ui

**Comment by lossposs** [comment]
> I now use heliboard instead, it has all these nice features, plus a better ui

You saved me tons of neurons appreciate

**Comment by habib12307** [comment]
Is it available now? I can't find it. 
Florisboard is best FOSS alternative to Gboard. But Currently I am using another one 🫤 and waiting for auto correction feature 😴

**Comment by lm41** [comment]
No, its the major goal for v0.6. See the [ROADMAP](https://github.com/florisboard/florisboard/blob/main/ROADMAP.md)
